ci/run_fuzzit.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -e
+
+git clone https://github.com/google/oss-fuzz.git
+export ENVOY_SRC_PATH=`pwd`
+cd oss-fuzz
+python infra/helper.py build_image --pull envoy
+python infra/helper.py build_fuzzers --sanitizer=address envoy ${ENVOY_SRC_PATH}
```

Comment:
One key issue here is build performance; if this takes a significant amount of time (which anecdotally it does from local runs), e.g. > 1 hour, it will become a CI bottleneck. Many of our CI jobs now benefit from Bazel caching and (in the future) RBE enablement, this one is going to be inherently stuck without this, since the oss-fuzz image doesn't have any plumbing for this. CC @lizan

Reply:
@htuch This is exactly what I'm trying to do just wanted to see if it works with the current oss-fuzz images. Anyway I was able to compile the targets with bazel for example

using `bazel build //test/server:server_fuzz_test` the build completes successfully but it seems that it doesn't link against libFuzzer. what would be the right command to link it against libfuzzer (i.e -fsanitize=fuzzer) as I don't wont to use the oss docker both to use the caching and also not to use the nested docker in Circle.

Reply:
Yeah, I'm afraid you're going to have to do some gnarly Bazel hacking to get it to link against libfuzzer under Bazel. We would be super appreciative of this work, but it will involve diving into https://github.com/envoyproxy/envoy/blob/master/bazel/envoy_test.bzl#L66 and figuring out how to cleanly offer both the corpus run (using the existing test driver) and a way to on the Bazel CLI configure a link against libfuzzer (and somehow getting libfuzzer included in the Envoy build).

Reply:
Thanks! I'll find the way to do that. I just wanted to make sure that I'm not doing double work and it's not implemented yet.